index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/src/assets/svg/logoTab.svg" />
+    <link rel="icon" type="image/svg+xml" href="/public/favicon.svg" />
```

Comment:
fix it

Files in the public directory are served at the root path.
Instead of /public/favicon.svg, use /favicon.svg.

Reply:
+ fixed